docs/tutorials/runtime-module-in-own-crate.md

```diff
@@ -0,0 +1,167 @@
+---
+title: "Write a Runtime Module as a Rust crate"
+---
+
+In this tutorial, you'll write a Substrate runtime module that lives in its own crate, and include it in a node based on the node-template. If you prefer you can follow along with the [video walkthrough](todo record after initial feedback).
+
+## Setup a Development Environment
+If you haven't already, follow the guide to [install substrate](getting-started/installing-substrate.md). You'll need the substrate-up scripts, but not the standard substrate node, so the `--fast` option is what you want here.
+
+## Create your node
+We'll begin by creating a brand new node template.
+```bash
+substrate-node-new modular-chain <your name>
```

Comment:
If the user follows this, then they need to either replace everything in the node template with `branch = 'v1.0` or they need to replace everything in the module template with `rev = <git-commit>`